lib/fathead/perl_doc/Parsers/parse-faq.pl

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env perl
```

Comment:
@jbarrett Hmm, you mean we have a separate routine for any (different) parsing, then a HASH from files (or groups) to the parser to be used to parse the page?